packages/webapp/package.json

```diff
@@ -10,6 +10,7 @@
         "test": "yarn run build"
     },
     "dependencies": {
+        "dayjs": "^1.10.4",
```

Comment:
dayjs <3 -- the days of moment are over 😢

Reply:
I know! Sad, right? Momentjs was my spacetime copilot for a long time!